libraries/AP_EFI/AP_EFI_Backend.h

```diff
@@ -31,6 +31,8 @@ class AP_EFI_Backend {
     // Update the state structure
     virtual void update() = 0;
 
+    virtual void handle_EFI_message(const mavlink_message_t &msg);
```

Comment:
```suggestion
    virtual void handle_EFI_message(const mavlink_message_t &msg) {};
```
No need to include in the cpp.